src/impl/chassis/controller/chassisControllerBuilder.cpp

```diff
@@ -413,6 +413,12 @@ ChassisControllerBuilder::buildDOCC(std::shared_ptr<ChassisController> chassisCo
 }
 
 std::shared_ptr<ChassisControllerPID> ChassisControllerBuilder::buildCCPID() {
+  auto ccGearset = gearset;
+  auto ccDriveScales = driveScales;
+  if(differentOdomScales) {
+    ccDriveScales = odomScales;
```

Comment:
Can't we just use `odomScales` in either case?

Reply:
Will test whether that works.